--- a/code/PT/print_tokens.c
+++ b/code/PT/print_tokens.c
@@ -228,7 +228,7 @@
                  case 16 : ch=get_char(tstream_ptr->ch_stream);
                            if(check_delimiter(ch)==TRUE)
                            {
-                                 token_ptr->token_id=keyword(next_st);
+                                 token_ptr->token_id=keyword(-next_st);
                                  unget_char(ch,tstream_ptr->ch_stream);
                                  token_ptr->token_string[0]='\0';
                                  return(token_ptr);
